### PR TITLE
feat: Make double tap and usage of IMU conditional

### PR DESCRIPTION
**Pull request by eng1n88r**
refactor: Improve code and remove some unsupported defines

### CODE REVIEW

**Review by Bentlybro — APPROVED** [review]
This works

File: src/BluetoothDeviceDriver/BluetoothDeviceDriver.ino [hunk 1]
```diff
@@ -139,10 +132,7 @@ void loop() {
   if (recording) {
     uint32_t available_samples = (recording_idx + BUFFER_SIZE - read_idx) % BUFFER_SIZE;
     if (available_samples >= CHUNK_SIZE) {
-      Serial.print("Sending ");
-      Serial.print(available_samples);
-      Serial.println(" samples");
-
+      Serial.println("Sending " + String(available_samples) + " samples");
```
Comment (axelpey) [inline src/BluetoothDeviceDriver/BluetoothDeviceDriver.ino:136]: Probably better to get rid of it. We've read writing to serial while sending BLE might cause issues. Would be better if you can put this behind a debug flag with a #define

**Review by axelpey — CHANGES_REQUESTED** [review]

File: src/BluetoothDeviceDriver/BluetoothDeviceDriver.ino (hunk 1 above)
Comment (eng1n88r) [reply src/BluetoothDeviceDriver/BluetoothDeviceDriver.ino:136]: Addressed your comment. Thank you for pointing that out!

**Review by axelpey — APPROVED** [review]
Good to me

**Pull request merged**